### DISCUSSION TITLE
Mountain Duck taking days to sync

### DISCUSSION THREAD

**Discussion by Dennywuh** [iterate-ch/cyberduck#16032] [Q&A]
Hi. I use Mountain Duck to sync hard drives on my desktop PC to buckets on Google Cloud Storage. The very first upload takes some time, logically, but the weekly sync usually completes in less than an hour.

I started the sync of a single folder yesterday, 19K files in total. It has now been running for 29 hours and there are still 13K files pending. Mountain Duck is using 60 % of the CPU and 38 % of the RAM.

Any idea what I'm doing wrong or what I can do to speed up the process?

Thanks, D.
_________________

Edit: Update

48 hours since it began syncing there are still 6K files pending. I've rebooted and there's no difference in speed. It takes each file approximately 10 seconds to sync.

Any suggestion will be greatly appreciated.

**Comment by tatze96** [comment]
Please write to support@mountainduck.io with the ticket reference GH-16032 for further investigation if the issue is still present. Ensure to include the following information:
* a [debug log](https://docs.cyberduck.io/mountainduck/support/#debug-log)
* Do you have the cache management settings (Purge files in cache not opened after, Limit cache size to) activated within the Mountain Duck preferences _Sync_ tab?
* Do you transfer large (>100MB) or small files?
* Which version of Mountain Duck is in use?
